public/app/features/transformers/spatial/SpatialTransformerEditor.tsx

```diff
@@ -122,7 +122,8 @@ export const SetGeometryTransformerEditor: React.FC<TransformerUIProps<SpatialTr
       props.onChange({ ...opts, ...props.options });
       console.log('geometry useEffect', opts);
     }
-  });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
@ryantxu is useEffect ok to run just once (on initialization of spatial transformer editor)?